numpy/_array_api/_searching_functions.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from ._array_object import ndarray
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from ._types import Tuple, array
+
+import numpy as np
+
+def argmax(x: array, /, *, axis: int = None, keepdims: bool = False) -> array:
```

Comment:
```suggestion
def argmax(x: array, /, *, axis: int | None = None, keepdims: bool = False) -> array:
```
`int` is proclaimed as the axis type here, yet its sentinel value is `None`.